utils/build_analyzers.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+
+# Build analyzers from Dockerfiles locally
```

Comment:
This script is made a bit redundant by the build_analyzers test suite... The idea was that you could use this script to, as quickly as possible (via concurrency), build all the docker images locally.